.github/workflows/inductor_xpu_e2e_nightly.yml

```diff
@@ -48,12 +53,11 @@ jobs:
         uses: actions/checkout@v4
       - name: Prepare Conda ENV
         run: |
-          which conda
-          if ! conda env list | grep -q "^e2e_ci "; then
-            conda create -n e2e_ci python=3.8 cmake ninja -y
-          fi
+          which conda && conda clean -ay
+          conda remove --all -y -n e2e_ci || rm -rf $(dirname ${CONDA_EXE})/../envs/e2e_ci
+          conda create -n e2e_ci python=${{ inputs.python }} cmake ninja -y
           source activate e2e_ci
-          conda install intel::mkl-static intel::mkl-include -y
+          conda install -c intel mkl-static mkl-include -y
```

Comment:
why we need this change?

Reply:
Ever use intel::mkl-static in CPU inductor test, it may be failed and don't know why.